src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/UpdateRecallController.kt

```diff
@@ -62,7 +62,8 @@ data class UpdateRecallRequest(
   val additionalLicenceConditions: Boolean? = null,
   val additionalLicenceConditionsDetail: String? = null,
   val differentNomsNumber: Boolean? = null,
-  val differentNomsNumberDetail: String? = null
+  val differentNomsNumberDetail: String? = null,
+  val recallNotificationEmailSentDateTime: LocalDate? = null
```

Comment:
Is this a date or date+time. If you need the date and the time in UTC, use `Instant` instead of `LocalDate`

Reply:
Actually, we are using OffsetDateTime everywhere else inc `recallEmailReceivedDateTime` in here. But agree it shouldn't be LocalDate as that is just a date, without the time.

Reply:
Ye, true. I changed it in the last commits. Just realised quite a lot of commits were not pushed ... silly